src/main/scala/io/iohk/ethereum/vm/DataWord.scala

```diff
@@ -98,4 +110,9 @@ class DataWord private (private val n: BigInt) extends Ordered[DataWord] {
     f"DataWord(0x$n%02x)" //would be even better to add a leading zero if odd number of digits
 
   def toBigInt: BigInt = n
+
+  /**
+    * @return Size in bytes excluding the leading 0 bytes
+    */
+  def byteSize: Int = if (isZero) 0 else (n.bitLength - 1) / 8 + 1
```

Comment:
It's just maths 😁 
For positive values it is equivalent to `1 + floor(log_256(n))`. See YP (222) (EXP gas cost) and the [unit tests](https://github.com/input-output-hk/etc-client/pull/46/files#diff-d7ae98f2e6380f5568a3e55bcbbd40a7R180)